bigbluebutton-html5/private/locales/en.json

```diff
@@ -580,16 +580,16 @@
     "app.video.stats.encodeUsagePercent": "Encode usage",
     "app.video.stats.currentDelay": "Current delay",
     "app.fullscreenButton.label": "Make {0} fullscreen",
-    "app.deskshare.iceConnectionStateError": "Error 1108: ICE connection failed when sharing screen",
-    "app.sfu.mediaServerConnectionError2000": "Error 2000: Unable to connect to media server",
-    "app.sfu.mediaServerOffline2001": "Error 2001: Media server is offline. Please try again later.",
-    "app.sfu.mediaServerNoResources2002": "Error 2002: Media server has no available resources",
-    "app.sfu.mediaServerRequestTimeout2003": "Error 2003: Media server requests are timing out",
-    "app.sfu.serverIceGatheringFailed2021": "Error 2021: Media server cannot gather ICE candidates",
-    "app.sfu.serverIceGatheringFailed2022": "Error 2022: Media server ICE connection failed",
-    "app.sfu.mediaGenericError2200": "Error 2200: Media server failed to process request",
-    "app.sfu.invalidSdp2202":"Error 2202: Client generated an invalid SDP",
-    "app.sfu.noAvailableCodec2203": "Error 2203: Server could not find an appropriate codec",
+    "app.deskshare.iceConnectionStateError": "Connection failed when sharing screen (ICE error 1108)",
+    "app.sfu.mediaServerConnectionError2000": "Unable to connect to media server (error 2000)",
+    "app.sfu.mediaServerOffline2001": "Media server is offline. Please try again later (error 2001)",
+    "app.sfu.mediaServerNoResources2002": "Media server has no available resources (error 2002)",
+    "app.sfu.mediaServerRequestTimeout2003": "Media server requests are timing out (error 2003)",
+    "app.sfu.serverIceGatheringFailed2021": "Media server cannot gather connection candidates (ICE error 2021)",
+    "app.sfu.serverIceGatheringFailed2022": "Media server connection failed (ICE error 2022)",
+    "app.sfu.mediaGenericError2200": "Media server failed to process request (error 2200)",
+    "app.sfu.invalidSdp2202":"Client generated an invalid join audio request (SDP error 2202)",
```

Comment:
Does this only apply to listen-only or does it also apply to webcams and screenshare?

Reply:
It applies to all three. Should be reworded to media request. This error almost never happens, tho.

Reply:
I see it used in webcams and screenshare code too, good catch! Adjusting